lingua_franca/lang/parse_en.py

```diff
@@ -580,7 +586,8 @@ def extractnumber_en(text, short_scale=True, ordinals=False):
                                         short_scale, ordinals).value
 
 
-def extract_duration_en(text):
+def extract_duration_en(text, resolution=DurationResolution.TIMEDELTA,
```

Comment:
This function now conflates two distinct ideas - durations and offsets. I would propose splitting this into two distinct functions - `extract_duration_en`, which always returns a `timedelta` (see my comment below about unit conversions), and `exract_offset_en` which always returns a `relativedelta`. The common logic can be extracted out into small, helper functions.

Note that this is basically the distinction that is made in `parse.py`, so being consistent here and using two functions for those two different concepts seems like it would make a lot of sense.